src/lib/store.js

```diff
@@ -335,11 +337,19 @@ export default class Store {
 
 	toggleConsentToolShowing = (isShown) => {
 		this.isConsentToolShowing = typeof isShown === 'boolean' ? isShown : !this.isConsentToolShowing;
+		this.isConsentToolShowing === true ? document.body.classList.add('cmp') : document.body.classList.remove('cmp');
 		this.isFooterShowing = false;
 		this.storeUpdate();
 	};
 
 	toggleFooterShowing = (isShown) => {
+		const vendorConsents = this.getVendorConsentsObject();
+		const publisherConsents = this.getPublisherConsentsObject();
+		if (this.isAllSetTrue(vendorConsents.purposeConsents) &&
+				this.isAllSetTrue(vendorConsents.vendorConsents) &&
+				this.isAllSetTrue(publisherConsents.standardPurposes)) {
```

Comment:
Aktualnie vendorConsents.purposeConsents i publisherConsents.standardPurposes to jest to samo

Reply:
Poprawione